src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine t1 = new Truck();
+        Machine b1 = new Bulldozer();
+        Machine e1 = new Excavator();
+
+        Machine[] machines = {t1, b1, e1};
+        for (Machine m : machines) {
+            m.doWork();
+            m.stopWork();
+        }
+    }
 
 }
```

Comment:
[Please don’t add redundant empty lines to your code.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop#please-dont-add-redundant-empty-lines-to-your-code)